airbyte-integrations/connectors/destination-bigquery/src/main/kotlin/io/airbyte/integrations/destination/bigquery/BigQueryDestination.kt

```diff
@@ -83,23 +96,189 @@ class BigQueryDestination : BaseConnector(), Destination {
                     .setUseLegacySql(false)
                     .build()
 
+            val result = BigQueryUtils.executeQuery(bigquery, queryConfig)
+
+            if (result.getLeft() == null) {
+                throw ConfigErrorException(result.right)
```

Comment:
@gisripa do you know offhand if this works the same way? (I'm not sure if throwing a config error correctly returns that error all the way to the ui, vs explicitly doing a `return AirbyteConnectionStatus().withStatus(FAILED)`)

Reply:
This was remnant from existing code. You are correct, this will blow it instead of returning failure. Maybe add result.right (which is job's error.status message) in `.withMessage()`

Reply:
Sure, I have changed the code to return a failure with an error message instead of throwing an exception

Reply:
Also removed the blank lines between variable declarations